libs/vkd3d/command.c

```diff
@@ -2691,12 +2669,6 @@ static void d3d12_command_list_clear_attachment_pass(struct d3d12_command_list *
             image_barrier_count, image_barriers));
     }
 
-    rendering_info.renderArea.offset.x = 0;
```

Comment:
Ah ... ye. splitting commits in rebases is not fun :(

Reply:
I don't know how to fix this at this point.

Reply:
*should* be gone now.